Workflow fails if report.json is missing for x86_64-linux system

During the execution of the review workflow (see `.github/workflows/review.yml`), the job for the `x86_64-linux` system sometimes completes with ":white_check_mark: *No rebuilds*", but then fails with the following error:

```
jq: error: Could not open file /home/runner/.cache/nixpkgs-review/pr-450515/report.json: No such file or directory
Error: Process completed with exit code 2.
```

This indicates that the workflow expects a report.json file to be present, but it may not be generated under some conditions (e.g., when there are no rebuilds or if an earlier step fails).

**Steps to Reproduce:**
1. Run the workflow on a PR that does not trigger rebuilds, or where nixpkgs-review fails to generate a report.json.
2. Observe that the workflow fails at the report generation step for x86_64-linux.

**Expected Behavior:**
- The workflow should handle missing report.json files gracefully and avoid failing with a jq error.
- If no rebuilds occur or the report cannot be generated, the workflow should log a clear message and continue if possible.
- If the report.json is critical, the workflow should surface the real root cause (e.g., why the file was not generated).

**Relevant workflow file:**
[.github/workflows/review.yml](https://github.com/Defelo/nixpkgs-review-gha/blob/69b2ac722d9afe9b728006e3a0c692fddc3cc676/.github/workflows/review.yml)

**Screenshot / Log:**
```
jq: error: Could not open file /home/runner/.cache/nixpkgs-review/pr-450515/report.json: No such file or directory
Error: Process completed with exit code 2.
```

**Suggested Fix:**
- Add a check for existence of report.json before calling jq, and handle the case where it is missing.
- Improve error reporting so that the cause of the missing file is clear.

--- a/.github/workflows/review.yml
+++ b/.github/workflows/review.yml
@@ -166,12 +166,12 @@ jobs:
           nixpkgs-review -- \
             pr ${PR_NUMBER} \
             --no-shell \
+            --no-exit-status \
             --no-headers \
             --print-result \
             --build-args="-L" \
             --pr-json="$PR_JSON" \
-            $EXTRA_ARGS \
-            || true
+            $EXTRA_ARGS
         working-directory: nixpkgs
         env:
           GITHUB_TOKEN: ${{ github.token }}

